package.json

```diff
@@ -11,7 +11,10 @@
   ],
   "main": "index.js",
   "scripts": {
-    "test": "mocha -R spec --timeout 10000 --require test/init.js test/*.test.js"
+    "pretest": "node pretest.js",
+    "lint": "eslint .",
+    "test": "mocha --timeout 10000 test/*.js",
```

Comment:
I think you'll need to restore the `--require test/init.js` because of how the `getDataSource` and `getDBConfig` globals are used in the tests.